default.nix

```diff
@@ -81,6 +81,17 @@ stdenv.mkDerivation rec {
       ocp-index
       utop
       ocamlformat
+      ppx_import
+      ppx_sexp_conv
+      ppx_deriving_yojson
+      ppx_hash
+      ppx_compare
+      sexplib
+      cmdliner
+      dune-build-info
+      menhir
+      uri
+      elpi
```

Comment:
What's up with all of this? Is this an accidental introduction linked to you needing these dependencies to prepare the overlays?

Reply:
Argh! This is a spurious commit, will fix, sorry for the noise.

Reply:
Removed.